### PR TITLE
VAGOV-2749 fix deployment error

### PR DESCRIPTION
**Pull request by mbenziane**

### CODE REVIEW

**Comment by mbenziane** [comment]
Created an environment devops-vagovcms#pr-270<br><br> [button_visit-site](http://internal-devops-vagovcms-3596-1403825296.us-gov-west-1.elb.amazonaws.com)

**Comment by mbenziane** [comment]
rebuild

**Comment by mbenziane** [comment]
Created an environment devops-vagovcms#pr-270<br><br> [button_visit-site](http://internal-devops-vagovcms-3637-1238505095.us-gov-west-1.elb.amazonaws.com)

**Comment by mbenziane** [comment]
rebuild

**Comment by mbenziane** [comment]
Created an environment devops-vagovcms#pr-270<br><br> [button_visit-site](http://internal-devops-vagovcms-3631-1201096040.us-gov-west-1.elb.amazonaws.com)

File: docker/config/engine/scripts/check-health-cms-service.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+while true
+do
+  STATUS=$(curl -s -o /dev/null -w '%{http_code}' http://localhost:$HTTP_PORT)
+  if [ $STATUS -eq $HTTP_CODE_STATUS ]; then
+    echo "Got 200 HTTP status code, CMS service is up."
+    break
+  else
+    echo "Got $STATUS HTTP status code, CMS Service is not up yet, retrying..."
+  fi
+  sleep $SLEEP_TIME
```
Comment (andyhawks) [inline docker/config/engine/scripts/check-health-cms-service.sh]: shouldn't this be under the `else`? we don't need to sleep if cms is up.

**Comment by andyhawks** [comment]
rebuild

**Comment by mbenziane** [comment]
Created an environment devops-vagovcms#pr-270<br><br> [button_visit-site](http://internal-devops-vagovcms-3469-2119275727.us-gov-west-1.elb.amazonaws.com)

**Pull request merged**